Assignment_05.rb

```diff
@@ -94,24 +167,25 @@ def initialize
     @tasks = []
   end
 
-  def add_one_task(task_attributes)
+  def add_task(task_attributes)
     task = Task.new(task_attributes[:id], task_attributes[:name], task_attributes[:time_logs])
+    self.check_id_uniqueness(task_attributes[:id])
```

Comment:
Can we move this uniqueness part inside validates_id method?

Reply:
@AritraChatterjee-qi instead of using class method, can we use instance method of task, like `task.valid?` should work if you are handing uniqueness also inside as part of validates_id method.

Reply:
@Ramyani-QI I don't think making it a class methods will help. The only way we can implement that is by making the task.valid? a parameterized method and passing the list of Tasks created so  far.

Reply:
Merging the PR after confirmation on call